.github/workflows/verify_source_urls.yml

```diff
@@ -0,0 +1,8 @@
+name: Verify stable source URLs
+on: [push]
```

Comment:
@meteorcloudy I think that replacing `push` with `pull_request` here should make the action run when external contributors open or push to PR branches.

Reply:
Oh, I see, thank you!